### PR TITLE
Moyo header

### PR DESCRIPTION
**Pull request by pvnovikova**
- [DEMO LINK](https://pvnovikova.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://pvnovikova.github.io/layout_moyo-header/report/html_report/)

 [x] Header height is set in 1 place (for the links)
 [x] Content is vertically centered (for any header height)
 [x] CSS is used to show all letters in Uppercase (don't type them in HTML)
 [x] Logo is an image wrapped with a link
 [x] CSS Variable is used for a blue color
 [x] Pseudo-element is used for a blue line below the active link
 [x] Code follows all the [Code Style Rules ❗️](https://github.com/pvnovikova/layout_moyo-header/blob/master/checklist.md)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1,3 +1,64 @@
...
+  margin: 0;
+  padding: 0;
+}
+
+header a img {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  height: 40px;
+}
```
Comment (etojeDenys) [inline src/style.css:36]: image

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,64 @@
...
+  color: #00acdc;
+}
+
+.nav__link:active {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
```
Comment (etojeDenys) [inline src/style.css:56]:
```suggestion
.nav__link:hover,
.nav__link:active,
.is-active {
  color: #00acdc;
}
```

Comment (etojeDenys) [reply src/style.css:56]: move this color to some var and reuse it everywhere

**Review by etojeDenys — CHANGES_REQUESTED** [review]
try to pass all tests, then re-request the review

or feel free to ask for some help in the chat

**Review by l4st1m0za — CHANGES_REQUESTED** [review]
Your test are still wierdly broken :( 
You can always write in FE chat, if you have anu difficulties!

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,84 @@
...
+  padding: 0 50px;
+
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header__logo {
+  display: flex;
+  height: 40px;
+  width: 40px;
```
Comment (GUSILLUS) [inline src/style.css:25]: you can move it to a variable

**Review by GUSILLUS — CHANGES_REQUESTED** [review]
GJ! Turn off draft mode, and fix a little bit codestyle

**Comment by pvnovikova** [comment]
@GUSILLUS please review https://github.com/mate-academy/layout_moyo-header/pull/4928. Thank you

**Pull request closed without merging**